src/main/java/sc/fiji/bdvpg/bdv/ScreenShotMaker.java

```diff
@@ -52,10 +52,7 @@
 import net.imglib2.util.Intervals;
 import net.imglib2.view.IntervalView;
 import net.imglib2.view.Views;
-import sc.fiji.bdvpg.bdv.projector.AccumulateAverageProjectorARGB;
-import sc.fiji.bdvpg.bdv.projector.AccumulateMixedProjectorARGB;
-import sc.fiji.bdvpg.bdv.projector.AccumulateSumProjectorARGB;
-import sc.fiji.bdvpg.bdv.projector.Projection;
+import sc.fiji.bdvpg.bdv.projector.*;
```

Comment:
We should avoid wildcard import. I know intellij automatically puts them, but that should be removed

Reply:
Done